ci/gitlab/.gitlab-ci.yml

```diff
@@ -183,7 +183,7 @@ xenial-pkg-test:
   stage: pkg-test
   script:
     - ./scripts/test_install_garage_deploy.sh
-    - ./scripts/test_install_aktualizr_and_update.sh
+    - ./scripts/test_install_aktualizr.sh
```

Comment:
Should we also run this script (`./scripts/test_install_aktualizr.sh`) as part of `bionic-pkg-test`?